src/app/Providers/components/VMwareProviderHostsTable/helpers.ts

```diff
@@ -2,5 +2,6 @@ import { IHostNetwork } from '@app/queries/types';
 
 export const formatHostNetwork = (network: IHostNetwork): string => {
   const { name, address, isDefault } = network;
-  return `${name} - ${address}${isDefault ? ' (default)' : ''}`;
+  const line = `${name} - ${address}${isDefault ? ' (default)' : ''}`;
```

Comment:
We should make "line" a more descriptive variable name so it is self documenting.

Reply:
Absolutely!